openmetadata-ui/src/main/resources/ui/src/utils/i18next/i18nextUtil.ts

```diff
@@ -13,15 +13,17 @@
 
 import { InitOptions } from 'i18next';
 import enUS from '../../locale/languages/en-us.json';
+import frFR from '../../locale/languages/fr-fr.json';
 
 // Returns i18next options
 export const getInitOptions = (): InitOptions => {
   return {
-    supportedLngs: ['en-US'],
+    supportedLngs: ['en-US', 'fr-FR'],
     resources: {
       'en-US': { translation: enUS },
+      'fr-FR': { translation: frFR },
     },
-    fallbackLng: ['en-US'],
+    fallbackLng: ['fr-FR'],
```

Comment:
```suggestion
    fallbackLng: ['en-US'],
```

Reply:
Thanks, switched it to french for testing and forgot to switch it back.